src/connectService.ts

```diff
@@ -197,10 +233,19 @@ class ConnectService extends IframeService {
    */
   async authCreate(accountId: string): Promise<Uint8Array> {
     const message = 'Add this account as a Ceramic authentication method'
+    const authSecret = (await this.authenticate(message)).slice(2)
+    this.storeAccount(accountId, authSecret)
+    return u8a.fromString(authSecret, 'base16')
```

Comment:
I added a `fromHex` utility function that could be used here instead of `fromString(..., 'base16')`: https://github.com/ceramicstudio/3id-connect/blob/develop/src/utils.ts#L5